grafana_monitoring/cloud_datasource.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: 1
+
+datasources:
+  - name: CloudInfluxDB
+    type: influxdb
+    access: proxy
+    uid: "openstack_grafana"
+    user: $PROD_INFLUX_USERNAME
+    url: "http://influx1.nubes.rl.ac.uk:8086"
```

Comment:
Could we have the URL be given by a variable, similar to how we use a variable for the user ?

Reply:
Yup will fix that now